_rules/element-lang-matches-unique-language-off6ek.md

```diff
@@ -0,0 +1,340 @@
+---
+id: off6ek
+name: HTML element language subtag matches language
+rule_type: atomic
+description: |
+  This rule checks that the primary language subtag of an element matches its language
+accessibility_requirements:
+  wcag20:3.1.2: # Language of Parts (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+  wcag-technique:H58: # Using language attributes to identify changes in the human language
+    forConformance: false
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: satisfied
+input_aspects:
+  - DOM tree
+  - Accessibility tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # using aria-labelledby instead
+  - 'alt-require'
+---
+
+## Applicability
+
+This rule applies to any HTML element for which all the following are true:
+
+- the element is an [inclusive descendant][] in the [flat tree][] of a `body` element; and
+- The element is in a [top-level browsing context][]; and
+- The element is in a [document][] with a [content type][] of `text/html`; and
+- The element has a `lang` [attribute value][] which is a [valid language tag][]; and
+- The element has some non-empty [text with the same programmatic language][].
+
+## Expectation
+
+For each test target, the [primary language][] of the [valid language tag][] is a [unique element language][].
```

Comment:
"a unique element language" is a strange concept in this sentence. Only after checking its definition I understood that you are checking that the primary language is the same language of the element.

```suggestion
For each test target, the [primary language][] of the [valid language tag][] is the [element language][].
```